packages/fx-core/resource/package.nls.json

```diff
@@ -1163,6 +1167,14 @@
   "driver.deploy.deployToAzureFunctionsDescription": "deploy the project to the Azure Functions.",
   "driver.deploy.deployToAzureStorageDescription": "deploy the project to the Azure Storages.",
   "driver.deploy.enableStaticWebsiteInAzureStorageDescription": "enable static website setting in Azure Storage.",
+  "driver.deploy.getStorageContainerError": "Failed to get storage container %s.",
+  "driver.deploy.checkContainerStaticWebsiteError": "Failed to check if the container %s enabled static website option.",
+  "driver.deploy.checkContainerStaticWebsiteRemoteError": "Failed to check if the container %s enabled static website option due to remote service error.",
+  "driver.deploy.enableStaticWebsiteError": "Failed to enable static website option for the container %s.",
+  "driver.deploy.enableStaticWebsiteRemoteError": "Failed to enable static website option for the container %s due to remote service error.",
+  "driver.common.suggestion.retryLater": "Please try again later.",
```

Comment:
'later' is vague, can it be more specific? like in x minutes?

Reply:
I can't give an exact time. This is because the service may be down at this time or the load may be too high to respond properly. Users will have to wait until the service is restored.

Reply:
Same here, you can use "try it again", which don't need to mention how long is "later".